documents/operating.md

```diff
@@ -21,10 +21,100 @@ SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
+6. Help maintain and further the BIFFUD public footprint.
+7. Try to attend BIFFUD events.
+
+### Elected Roles
+In addition to base expectations, the following positions are to be held
+by corporate overlords.  They are elected annually:
+
+- **President:** Has a gavel.  The President is responsible for merging
+approved pull requests to `\corporate`.
+
+- **Vice President:** Has a smaller gavel.  The Vice President is responsible
+for laughing whenever the President makes a joke.
+
+- **Treasurer:** Keeps track of money, works with accountants, oversees
+and approves small expenses.
+
+- **Secretary:** Ensures notes and minutes are taken for meetings,
+ensures that proper CONTRIBUTING.md protocols are followed in `/corporate`.
+
+- **Beastmaster:** Ensures that the corporate overlords are aware of the
+progress and activity of BIFFUD projects.
+
+These roles are responsible for overseeing their duties, though in some
+cases that responsibility can take the form of setting up processes and
+support networks.  For instance, the Beastmaster can delegate oversight
+over a subset of BIFFUD projects if we have too many.
+
+### Adding Corporate Overlords
+Although previously thought impossible, an ancient ritual found buried
+in the ruins of the Lawyer Cave describes the process of creating new
+BIFFUD Corporate Overlords.
+
+Any Corporate Overlord may nominate the addition of a new Overlord during
+a plotting session.  If the nomination is seconded, the ritual begins.
+
+1. The Corporate Overlords are presented with a biography of the candidate.
+2. The candidate submits a piece of original poetry or other creative work
+for the Overlords' enjoyment and consideration.
+3. A vote is held.
+
+If 70% or more of the Corporate Overlords approve the addition, then after the
+plotting session ends the new Corporate Overlord is onboarded and ownership of
+the company is redistributed so that each overlord owns an equal share.
+
+
+### Removing Corporate Overlords
+A Corporate Overlord can recuse itself from BIFFUD at any time by dying, uploading
+its consciousness to the cloud, or submitting a letter of resignation to both
+the President of BIFFUD and the President of the United States.
+
+Corporate Overlords can also vote to remove a peer from the organization if
+any of the following conditions apply:
+
+- The Corporate Overlord fails to attend 50% of the monthly plotting sessions
+in a given year without a note from a family member explaining its absence.
+- The Corporate Overlord has violated a serious federal law in the past year.
+- The Corporate Overlord is hated or despised.
+
+The removal of a corporate overlord can be nominated during a plotting session
+and requires 70% approval or more from the other Corporate Overlords.
+
+Overlords are entitled to a pro-rated profit share for their tenure in a given
+year, as well as ongoing profit-share according to their ownership of various
+BIFFUD projects.
```

Comment:
Also, the same seemingly arbitrary 70% pops up here, but maybe that's fine.

Also also, you should add in that the nomination for removal needs to be seconded, just like with adding.